src/app/item-popup/PlugTooltip.tsx

```diff
@@ -30,17 +30,15 @@ export default function PlugTooltip({
 
   // display perk's synergy with masterwork stat
   const synergyStat =
-    item.masterworkInfo &&
-    item.masterworkInfo.statHash &&
+    idx(item, (i) => i.masterworkInfo.statHash) &&
     plug.plugItem.investmentStats &&
     plug.plugItem.investmentStats.some(
       (stat) =>
         stat.value > 0 &&
         stat.statTypeHash &&
-        item.masterworkInfo &&
-        item.masterworkInfo.statHash === stat.statTypeHash
+        idx(item, (x) => x.masterworkInfo.statHash) === stat.statTypeHash
     ) &&
-    ` (${item.masterworkInfo.statName})`;
+    ` (${idx(item, (i) => i.masterworkInfo.statName)})`;
```

Comment:
I think it can be known that `item.masterworkInfo` is not null here.

Reply:
reverted to more traditional here, so typescript did not need to revalidate same thing's existence further down